drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -75,11 +79,10 @@ static int bq274xx_ctrl_reg_write(const struct device *dev, uint16_t subcommand)
 	const struct bq274xx_config *config = dev->config;
 	int ret;
 
-	uint8_t tx_buf[3] = {
-		BQ274XX_CMD_CONTROL_LOW,
-		subcommand & 0xff,
-		subcommand >> 8,
-	};
+	uint8_t tx_buf[3];
+
+	tx_buf[0] = BQ274XX_CMD_CONTROL_LOW;
+	sys_put_le16(subcommand, &tx_buf[1]);
```

Comment:
Very nice.
```suggestion
	tx_buf[0] = BQ274XX_CMD_CONTROL;
	sys_put_le16(subcommand, &tx_buf[1]);
```
It looks like `BQ274XX_CMD_CONTROL_HIGH` can go.

Reply:
Yes! Done.